src/components/Calculator.js

```diff
@@ -1,53 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-export default class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      numbers: ['AC', '+/-', '%', '7', '8', '9', '4', '5', '6', '1', '2', '3', '0', '.'],
-      operators: ['÷', 'x', '-', '+', '='],
-      total: '0',
-      next: null,
-      operation: null,
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to fix the display of your calculator to accommodate failed modulus operation.
- [ ] **[OPTIONAL]** The live demo link in your README returns a 404 error. It would be nice to fix that or add a placeholder instead.
- [ ] **[OPTIONAL]** Your README paragraph contains a typo. I think you meant **First**, not **Frist**.